src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -74,16 +73,16 @@
     private Map<String, EntryType> entryTypes;
     private boolean eof;
     private int line = 1;
-    private final FieldContentParser fieldContentParser = new FieldContentParser(
-            FieldContentParserPreferences.fromPreferences(Globals.prefs));
+    private final FieldContentParser fieldContentParser;
     private ParserResult parserResult;
     private static final Integer LOOKAHEAD = 64;
     private final Deque<Character> pureTextFromFile = new LinkedList<>();
+    private final ImportFormatPreferences importFormatPreferences;
 
-
-    public BibtexParser(Reader in) {
+    public BibtexParser(Reader in, ImportFormatPreferences importFormatPreferences) {
```

Comment:
I think the whole `ImportFormatPreferences` contains to much information for the parser. In the end only the following information is needed:
- keyword separator
- list of wrapped fields
Thus I would prefer if only these two are passed as parameters.
(One can argue that both data should actually be stored in the bib-metadata. )